src/Common/src/TypeSystem/Common/DefType.FieldLayout.cs

```diff
@@ -2,35 +2,81 @@
 // Licensed under the MIT license. See LICENSE file in the project root for full license information.
 
 using System;
+using System.Diagnostics;
 
 namespace Internal.TypeSystem
 {
     // This is the api surface necessary to query the field layout of a type
-
     public abstract partial class DefType : TypeDesc
     {
+        private class FieldLayoutFlags
+        {
+            public const int HasContainsPointers = 1;
```

Comment:
This is just moving existing code between files. Are you ok with considering that for a future change to keep the diffs clean?